CMakeLists.txt

```diff
@@ -10,41 +10,68 @@
 
 # Preamble
 
-cmake_minimum_required(VERSION 3.17 FATAL_ERROR)
+cmake_minimum_required(
+  VERSION 3.17
+  FATAL_ERROR)
 
 # it's important to specify accurately the list of languages. for instance C and
 # C++ as we _do_ have some C files to compile explicitely as C (e.g. gl3w.c)
-project(O2 LANGUAGES C CXX VERSION 1.2.0)
+project(
+  O2
+  LANGUAGES C
+            CXX
+  VERSION 1.2.0)
 
 include(CTest)
 
 # Project wide setup
 
 # Would better fit inside GPU/CMakeLists.txt, but include GPU/Common directly
-set(ALIGPU_BUILD_TYPE "O2")
-
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
-set_property(GLOBAL PROPERTY REPORT_UNDEFINED_PROPERTIES)
-
-cmake_host_system_information(RESULT _totalmem QUERY TOTAL_PHYSICAL_MEMORY)
-math(EXPR _total_analysis_jobs "(${_totalmem}-4096)/10240")
-if(_total_analysis_jobs LESS_EQUAL 0)
-  set(_total_analysis_jobs 1)
+set(ALIGPU_BUILD_TYPE
+    "O2")
+
+set(CMAKE_MODULE_PATH
+    ${CMAKE_MODULE_PATH}
+    ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+set_property(
+  GLOBAL
+  PROPERTY REPORT_UNDEFINED_PROPERTIES)
```

Comment:
Sorry, but this (and many other) changes are completely unreadable. Who decided this was a good idea?

Reply:
Fixed in https://github.com/AliceO2Group/AliceO2/pull/6203/commits/22c0f8ee854b52b6920c9a71484e0f6772dc9be1
The only change in the configuration now is `max_pargs_hwrap = 5` -> `max_pargs_hwrap = 1` in order to preserve the current vertical style of `SOURCES`.